index.js

```diff
@@ -0,0 +1,45 @@
+const fs = require('fs');
+const { Client, Collection, Intents } = require('discord.js');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/EvilInsultGenerator/discord-bot/pullRequest?prid=8755514)
```suggestion
const { Client, Collection, Intents } = require("discord.js");
```